src/main/java/org/jenkinsci/plugins/workflow/cps/global/WorkflowLibRepositoryListener.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Sorry to be a pest but can you use the MIT license please, as specified in the POM?